tests/integration/targets/npm/tasks/no_bin_links.yml

```diff
@@ -0,0 +1,64 @@
+---
+- name: 'Remove any node modules'
+  file:
+    path: '{{ remote_dir }}/node_modules'
+    state: absent
+
+- vars:
+    # sample: node-v8.2.0-linux-x64.tar.xz
+    node_path: '{{ remote_dir }}/{{ nodejs_path }}/bin'
+    package: 'thelounge'
```

Comment:
I should probably change this package to a simpler package which creates `.bin` folder, but for now this works :)

Reply:
Yeah, that would be great. I searched a bit on npm, and found https://www.npmjs.com/package/ncp which has no dependencies and installs a CLI program.

Reply:
Thank you very much for your suggestion, it fixed all tests as well.